cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-items.cy.js

```diff
@@ -108,14 +111,17 @@ describe('bulk-edit', () => {
       BulkEditSearchPane.uploadFile(validItemAccessionNumbersFileName);
       BulkEditSearchPane.waitFileUploading();
 
+      const newLocation = 'Online';
```

Comment:
maybe better named this onlineLocation and annexLocation?

Reply:
May be yes, but in general it doesn't matter because I just replace old location for new one)